stripe/src/main/java/com/stripe/android/model/CountryCode.kt

```diff
@@ -0,0 +1,25 @@
+package com.stripe.android.model
+
+import android.os.Parcelable
+import androidx.annotation.Size
+import kotlinx.parcelize.Parcelize
+import java.util.Locale
+
+@Parcelize
+data class CountryCode(
+    @Size(2) private val letters: String,
+    val twoLetters: String = letters.toUpperCase(Locale.getDefault())
+) : Parcelable {
+    internal fun isUS() = twoLetters == "US"
+    internal fun isCA() = twoLetters == "CA"
+    internal fun isGB() = twoLetters == "GB"
+
+    companion object {
+        val US = CountryCode("US")
+        val CA = CountryCode("CA")
+        val GB = CountryCode("GB")
+        fun isUS(countryCode: CountryCode?) = countryCode?.isUS() ?: false
```

Comment:
`countryCode` == `CountryCode.US` (and same for others)

Reply:
👍